### ISSUE TITLE
Limit number of pair matched elements 

### ISSUE THREAD

**Issue by skindsc1** [spawaskar-cora/cora-docs#107]
It would be helpful to be able to limit the number of pair matched elements that can be added to one.  I was able to add an element to the pair match list when there had already been an element listed as a pair match.

![image](https://user-images.githubusercontent.com/21186879/38560060-60251c7a-3c9a-11e8-95e3-d8b18b11d8f4.png)

**Comment by SachinPawaskarUNO** [comment]
@nmcelroy and @rpernst to enhance the pair match screen based on our discussion today.

**Comment by clegarde** [comment]
Agreed that we do need the option to PM to more than one incomplete bone. We could consider having some sort of error or warning if you try to pair match a complete bone to two other _complete_ bones? It should be able to 
- 1 Complete bone to 1+ incomplete
Or
- 1 complete to 1 complete
Should NOT be
- 1 complete to 2+ complete bones

Not sure how complicated that would be and if necessary? @SachinPawaskarUNO we could discuss this, but I don't think this is entirely necessary. Could be nice to have.
